modules/vba_documents/app/swagger/vba_documents/v1/description.md

```diff
@@ -56,5 +56,11 @@ A request to the `/uploads/{id}` endpoint will return a real-time status for tha
 
 The `updated_at` field indicates the last time the status for a given GUID was updated.
 
+### Optional Base64 Encoding
+
+Base 64 is an encoding scheme that converts binary data into text format so that encoded textual data can be easily transported over network un-corrupted and without any data loss. Base64 is used to take our binary multipart/form-data and encode it in it's entirety in order to bypass certain false positives encountered when going through the VA Firewall. If you feel like you are getting a greater number of PDF files being rejected when uploading them through the PUT request then you might consider encoding the entire payload in base64.
+
+After encoding your payload you'll want to preface your base64 string with `data:application/pdf;base64,` in order to allow our system to distinguish the type of file you upload so your final string payload would look something like `data:application/pdf;base64,(encryption string)==` making sure that at the end of your payload you close it off with the standard == marker.
```

Comment:
I wouldnt say they want to add the data:application/pdf;base54. They need to or it wont work

Reply:
I'll make it required